docs/post_installation/post_installation.md

```diff
@@ -140,19 +142,14 @@ HTTP REST API related options exists under the section `restAPI` within the `con
   }
 ```
 
-If you want to make the HTTP REST API only accessible from localhost, change the `restAPI.bindAddress` config option
-accordingly.
+If you want to make the HTTP REST API only accessible from localhost, you change the `restAPI.bindAddress` config option accordingly.
 
-`restAPI.permittedRoutes` defines which routes can be called from foreign addresses which are not defined under
-`restAPI.whitelistedAddresses`.
+`restAPI.permittedRoutes` defines which routes can be called from foreign addresses which are not defined under `restAPI.whitelistedAddresses`.
 
-If you are concerned with resource consumption, consider turning off `restAPI.powEnabled`, which makes it so that
-clients must perform Proof-of-Work locally, before submitting a message for broadcast. In case you'd like to offer
-Proof-of-Work for clients, consider upping `restAPI.powWorkerCount` to provide a faster message submission experience.
+If you are concerned with resource consumption, consider turning off `restAPI.powEnabled`.  This way, the clients must perform Proof-of-Work locally, before submitting a message for broadcast. In case you'd like to offer Proof-of-Work to clients, consider increasing the `restAPI.powWorkerCount` to provide a faster message submission experience.
```

Comment:
```suggestion
If you are concerned with resource consumption, consider turning off `restAPI.powEnabled`.  This way, the clients must perform proof of work locally, before submitting a message for broadcast. In case you'd like to offer proof of work to clients, consider increasing the `restAPI.powWorkerCount` to provide a faster message submission experience.
```

Reply:
So proof of work is written out as "proof of work", "Proof-of-Work", or "POW" in a lot of our docs; I changed these ones to reflect where you mentioned POW earlier in your edits so it was consistent. Don't know if you want to switch it up later but I just wanted to draw your attention to it!